### PR TITLE
Add autoblack to workflow

### PR DESCRIPTION
**Pull request by htoftevaag**
This addition will make sure all code that is not blackened will be automatically on pull

### CODE REVIEW

**Comment by htoftevaag** [comment]
It would be nice to check if it actually works after merging :)

File: .github/workflows/autoblack.yml [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+        run: black --check .
+      - name: If needed, commit black changes to the pull request
+        if: failure()
+        run: |
+          black .
+          git config --global user.name 'htoftevaag'
+          git config --global user.email 'htoftevaag@users.noreply.github.com'
+          git remote set-url origin https://x-access-token:${{ secrets.Autoblack_access_token }}@github.com/$GITHUB_REPOSITORY
+          git checkout $GITHUB_HEAD_REF
+          git commit -am "fixup: Format Python code with Black"
```
Comment (gomezzz) [inline .github/workflows/autoblack.yml:29]: Can we change the commit message to

"WORKFLOW: Format Python code with black"

to make it clear that these are automatic commits?

**Review by gomezzz — CHANGES_REQUESTED** [review]

File: .github/workflows/autoblack.yml [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+on: [pull_request]
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v1
+      - name: Set up Python 3.7
+        uses: actions/setup-python@v1
+        with:
+          python-version: 3.7
```
[the remark this answers is not in the document]
Comment (htoftevaag) [reply .github/workflows/autoblack.yml:16]: Agreed.

File: .github/workflows/autoblack.yml (hunk 1 above)
Comment (htoftevaag) [reply .github/workflows/autoblack.yml:29]: Yes

**Review by gomezzz — APPROVED** [review]

**Pull request merged**